from https://salsa.debian.org/debian-gps-team/pkg-gpsd/-/blob/63115c01176f7e22aa74041d30bb20b4c72aa575/debian/patches/gpsd_hotplug_rules_disable_generic_serial_converters

Disable hotplugging of generic usb<>serial adapters.
These adapters are too common to hit them with the gpsd hotplug script.
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=550964
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=696020
Index: pkg-gpsd/gpsd.rules.in
===================================================================
--- pkg-gpsd.orig/gpsd.rules.in	2026-01-17 17:24:11.898915583 +0100
+++ pkg-gpsd/gpsd.rules.in	2026-01-17 17:24:11.896956600 +0100
@@ -91,10 +91,12 @@
 
 # Cygnal Integrated Products, Inc. CP210x Composite Device
 # (Used by Holux m241 and Wintec grays2 wbt-201) [linux module: cp210x]
-ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="ea60", SYMLINK+="gps%n", @UDEVCOMMAND@
+# !!! rule disabled in Debian because it matches way too many generic serial converters
+#ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="ea60", SYMLINK+="gps%n", @UDEVCOMMAND@
 
 # Cygnal Integrated Products, Inc. [linux module: cp210x]
-ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="ea71", SYMLINK+="gps%n", @UDEVCOMMAND@
+# !!! rule disabled in Debian because it matches way too many generic serial converters
+#ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="ea71", SYMLINK+="gps%n", @UDEVCOMMAND@
 
 # Cypress M8/CY7C64013 (Delorme uses these) [linux module: cypress_m8]
 ATTRS{idVendor}=="1163", ATTRS{idProduct}=="0100", SYMLINK+="gps%n", @UDEVCOMMAND@
